### PR TITLE
[SAIC-239] Fix requirements.txt for download custom paramiko

### PR DESCRIPTION
**Pull request by mirrorcoder**
This patch download custom paramiko from my github-repo, because native paramiko has bug(race condition)
Issue - https://github.com/paramiko/paramiko/issues/570
Pull - https://github.com/paramiko/paramiko/pull/571

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -17,3 +17,4 @@ pyyaml
 redis
 sqlalchemy
 oslo.config==1.15.0
++-e git://github.com/mirrorcoder/paramiko.git#egg=paramiko
```
Comment (roman-verchikov) [inline requirements.txt]: Please add comments describing why we need this (link to paramiko bug would probably do the job)

**Comment by roman-verchikov** [comment]
Please fix unittests

**Comment by roman-verchikov** [comment]
Please describe the patch in commit message. Commit message should contain paramiko bug link and high-level bug description (may be taken from Jira ticket)

**Comment by roman-verchikov** [comment]
+1

**Comment by realsystem** [comment]
+1 and make sure that func tests are OK

**Comment by roman-verchikov** [comment]
Most recent functional test run succeeded with this patch, can be merged.

**Pull request merged**